### PR TITLE
update marketplace latest release from picked develop branch

### PR DESCRIPTION
**Pull request by Joeper214**

### CODE REVIEW

**Comment by Joeper214** [comment]
Please review and Merge PR. @root-beer  🙇

**Comment by root-beer** [comment]
I'll review now.

**Comment by Joeper214** [comment]
よるしくおねがいします。

**Comment by root-beer** [comment]
Some Japanese translations has been deleted and it is not displayed properly.

File: config/locales/ja.yml [hunk 1]
```diff
@@ -730,23 +708,3 @@ ja:
     incrementHour: '1時間後'
     decrementHour: '1時間前'
     pickHour: '時間を選択してください'
-  instructions:
```
Comment (Joeper214) [inline config/locales/ja.yml]: 😨 Maybe its in these lines?

Comment (root-beer) [inline config/locales/ja.yml]: Yes. Thanks!

**Comment by root-beer** [comment]
LGTM! Thanks. :bow:

**Comment by Joeper214** [comment]
Let's release awe-marketplace branch? @root-beer

**Comment by root-beer** [comment]
:ok_man:

**Pull request merged**